File: frontend/lib/dtos/user_field_update_request.dart
class UserFieldUpdateRequest {
  final String fieldName;
  final String newValue;

  UserFieldUpdateRequest({
    required this.fieldName,
    required this.newValue,
  });

  Map<String, dynamic> toJson() => {
        'fieldName': fieldName,
        'newValue': newValue,
      };
}

File: backend/src/main/java/com/toprakrehberi/backend/models/UserFieldUpdateRequest.java
package com.toprakrehberi.backend.models;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class UserFieldUpdateRequest {
    private String fieldName;
    private String newValue;

}
